packages/wallet/src/wallet.ts

```diff
@@ -521,7 +521,7 @@ export class Wallet extends Signer {
 
     const [txs, n] = await Promise.all([
       this.buildUpdateConfigTransaction(config, publish, indexed),
-      nonce ? nonce : await this.getNonce()
+      nonce ?? this.getNonce()
```

Comment:
BTW, this actually changes behaviour, so please let me know if this is OK. Before, if `nonce === 0`, this would make an async call to `this.getNonce()`. Now, it actually return 0 directly if `nonce === 0`.

Reply:
This may be better actually, but I'll let @Agusx1211 confirm as well.